### ISSUE TITLE
Unsynced data not available offline

### ISSUE THREAD

**Issue by kevin3721** [keybase/client#16220]
Keybase GUI Version: 3.0.0-20190205104115+6394d03e6c

**Comment by strib** [comment]
This will be fixed in the next release, probably.  But if you want to do a `keybase log send` I can take a look to make sure it's the same thing we fixed.

**Comment by kevin3721** [comment]
yeah, I'd like to do a keybase lod but I don't know how to do it.

**Comment by strib** [comment]
In a terminal or command prompt, depending on what platform you're on.  If you're on mobile, it's the "Send feedback" option from the menu.

**Comment by kevin3721** [comment]
How about the windows app?

**Comment by strib** [comment]
Open the "Command prompt" program, type `keybase log send`, hit enter, then type `YES` at the first question and hit enter.  Then type a description of what you experienced, and hit enter twice.

**Comment by kevin3721** [comment]
Thanks. Sent.

**Comment by DonaldKBrown** [comment]
I am also experiencing this error. Desktop client in Ubuntu 18.04 when creating a private repo through the GUI and CLI on Ubuntu 18.04 Server when trying to push to that private repo.

**Comment by strib** [comment]
Should be fixed in the next release, coming out in a day or two.

**Comment by bjmgeek** [comment]
I'm also experiencing this error with the current version on Windows 10.
